### PR TITLE
Adding mxnet inference for squeezenet model

### PR DESCRIPTION
**Pull request by goswamig**
*Description of changes:*
Adding Squeezenet model for inference using MMS.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: samples/mnist/inference/mxnet/squeezenet.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        app: squeezenet-service
+    spec:
+      containers:
+      - name: squeezenet-service
+        image: 763104351884.dkr.ecr.us-east-1.amazonaws.com/mxnet-inference:1.4.0-gpu-py36-cu90-ubuntu16.04
+        args:
+        - mxnet-model-server
+          --start
+          --mms-config /home/model-server/config.properties
+          --models squeezenet=https://s3.amazonaws.com/model-server/models/squeezenet_v1.1/squeezenet_v1.1.model
```
Comment (vdantu) [inline samples/mnist/inference/mxnet/squeezenet.yaml:38]: This line, the squeezenet-v1.1.model should be changed to "mnist_cnn.mar" path.

File: docs/inference/mxnet/mxnet.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+   ```
+
+   ```
+   curl -X POST http://127.0.0.1:8080/predictions/squeezenet -T kitten.jpg 
+   ```
+
+   The result should be something like below.
+
+   ```
+   Handling connection for 8080
```
Comment (vdantu) [inline docs/inference/mxnet/mxnet.md:54]: This output needs to change to what "mnist_cnn.mar" file returns.

**Review by vdantu — COMMENTED** [review]
If these instructions work for MMS+K8s+Squeezenet_v1.1, then changing the ".model" file path to "mnist_cnn.mar" path in this repository will make these instructions work for "MMS+K8s+mnist_cnn.mar"

**Comment by goswamig** [comment]
@vdantu  please make the appropriate changes according to mnist model in this file.

**Comment by arun-gupta** [comment]
Will close this PR as MXNet inference is now working and documented clearly. However still need to have these steps running on EKS as mentioned in https://github.com/aws-samples/machine-learning-using-k8s/issues/88.

**Pull request closed without merging**